src/auth_components/PrivateRoute.tsx

```diff
@@ -6,9 +6,8 @@ export default function PrivateRoute({ component: Component, ...rest }: any) {
 
   return (
     <Route
-      {...rest}
-      render={(props) => {
-        return currentUser ? <Component {...props} /> : <Redirect to="#" />;
+      render={() => {
+        return currentUser ? <Component {...rest} /> : <Redirect to="#" />;
```

Comment:
This change seems to work fine. I am not sure actually why the previous code was passing "props"? Basically lines 9-10 of the old code I don't understand, @mmcknett maybe you have insight? 😊

Reply:
The existing code seems like it follows the [example in the documentation](https://v5.reactrouter.com/web/api/Route/render-func) for when you use the `render` prop of `Route`. That's the best I got. 😅